Documentation/coding-guidelines/interop-pinvokes.md

```diff
@@ -95,43 +95,51 @@ Guids are usable directly in signatures. When passed by ref they can either be p
+|:------|:-----------------|:-------------------|:---------|:-------------------------------------|
+| 32    | `BOOL`           | `int`              | `int`    | `bool`                               |
+| 8     | `BOOLEAN`        | `unsigned char`    | `byte`   | `[MarshalAs(UnmanagedType.U1)] bool` |
+| 8     | `BYTE`           | `unsigned char`    | `byte`   |                                      |
+| 8     | `CHAR`           | `char`             | `sbyte`  |                                      |
+| 8     | `UCHAR`          | `unsigned char`    | `byte`   |                                      |
+| 16    | `SHORT`          | `short`            | `short`  |                                      |
+| 16    | `CSHORT`         | `short`            | `short`  |                                      |
+| 16    | `USHORT`         | `unsigned short`   | `ushort` |                                      |
+| 16    | `WORD`           | `unsigned short`   | `ushort` |                                      |
+| 16    | `ATOM`           | `unsigned short`   | `ushort` |                                      |
+| 32    | `INT`            | `int`              | `int`    |                                      |
+| 32    | `LONG`           | `long`             | `int`    |                                      |
+| 32    | `ULONG`          | `unsigned long`    | `uint`   |                                      |
+| 32    | `DWORD`          | `unsigned long`    | `uint`   |                                      |
+| 64    | `QWORD`          | `__int64`          | `long`   |                                      |
+| 64    | `LARGE_INTEGER`  | `__int64`          | `long`   |                                      |
+| 64    | `LONGLONG`       | `__int64`          | `long`   |                                      |
+| 64    | `ULONGLONG`      | `unsigned __int64` | `ulong`  |                                      |
+| 64    | `ULARGE_INTEGER` | `unsigned __int64` | `ulong`  |                                      |
+| 32    | `HRESULT`        | `long`             | `int`    |                                      |
+| 32    | `NTSTATUS`       | `long`             | `int`    |                                      |
+
+
+The following types, being pointers, do follow the width of the platform. Use `IntPtr`/`UIntPtr` for these.
+
+| Signed Pointer Types (use `IntPtr`) | Unsigned Pointer Types (use `UIntPtr`) |
+|:------------------------------------|:---------------------------------------|
+| `HANDLE`                            | `WPARAM`                               |
+| `HWND`                              | `UINT_PTR`                             |
+| `HINSTANCE`                         | `ULONG_PTR`                            |
+| `LPARAM`                            | `SIZE_T`                               |
+| `LRESULT`                           |                                        |
+| `LONG_PTR`                          |                                        |
```

Comment:
Perhaps worth mentioning here: one interesting `typedef` for `LONG_PTR` is `SSIZE_T` in `BaseTsd.h`, that has same named counterpart is Unix in small casing `ssize_t`. Both are 'width-wise platform-dependent', signed types.

From `BaseTsd.h`:
> typedef LONG_PTR SSIZE_T, *PSSIZE_T;

Reply:
Perhaps you'd like to offer a similar table for Unix?